src/Affects/URL/Events/ConfigureURL.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of the tenancy/tenancy package.
+ *
+ * (c) Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see http://laravel-tenancy.com
+ * @see https://github.com/tenancy
+ */
+
+namespace Tenancy\Affects\URL\Events;
+
+use Tenancy\Identification\Events\Switched;
+use Illuminate\Contracts\Routing\UrlGenerator;
+
+class ConfigureURL
+{
+    /**
+     * @var Switched
+     */
+    public $event;
+    /**
+     * @var UrlGenerator
+     */
+    public $url;
+
+    public function __construct(Switched $event, UrlGenerator $url)
+    {
+        $this->event = $event;
+        $this->url = $url;
+    }
+
+    public function changeUrlTo(string $url)
```

Comment:
I prefer simpler methods, the way I read it is like this:

ConfigureURL::changeUrlTo

Two things, URL vs Url and having them twice. So my suggestion would be:

ConfigureURL::forceRoot or something?

Reply:
I've been thinking about that Url vs URL as well. Not only in the function name but the entire naming for the namespace, classes etc.
I personally would go for URL all the way, Laravel uses Url throughout class and function names though.
It's literally a question of preference in my opinion.

I like the "change" rather than the force, so would make it ConfigureURL::changeRoot, or maybe just ConfigureURL::change

Reply:
changeRoot to make it explicit?